avaframe/runCom4FlowPy.py

```diff
@@ -70,36 +76,51 @@ def main():
         # Create result directory
         # NOTE-TODO: maybe move into separate function as well ...
         timeString = datetime.now().strftime("%Y%m%d_%H%M%S")
-        cfgPath["resDir"] = cfgPath["outDir"] / "res_{}".format(timeString)
-        fU.makeADir(cfgPath["resDir"])
-        cfgPath["tempDir"] = cfgPath["workDir"] / "temp"
-        fU.makeADir(cfgPath["tempDir"])
+
+        cfgPath["resDir"] = cfgPath["outDir"] / "peakFiles" / "res_{}".format(uid)  # (timeString)
+        # check if simulation with same uid already has results folder
+        if os.path.isdir(cfgPath["resDir"]):
+            log.info("folder with same name already exists - aborting")
+            log.info("simulation results folder with same .ini parameters already exists")
+            sys.exit(1)
+        else:
+            fU.makeADir(cfgPath["resDir"])
+            cfgPath["tempDir"] = cfgPath["workDir"] / "temp"
+            fU.makeADir(cfgPath["tempDir"])
 
         cfgPath["deleteTemp"] = "False"
 
+        cfgPath["uid"] = uid
+        cfgPath["timeString"] = timeString
+        cfgPath["outputFiles"] = cfgCustomPaths["outputFiles"]
+
         com4FlowPy.com4FlowPyMain(cfgPath, cfgSetup)
```

Comment:
does it make sense to put this line outside the if- part?

Reply:
```cfgPath``` is initialized inside the if ```if cfgCustomPaths["useCustomPaths"] == ``` statements - slightly different depending on the choice of ```"useCustomPaths"```, so it makes sense to have these statements twice - also ```uid``` is calculated slightly different (in case of ```"useCustomPaths" == "False"``` also the AvalancheDir from the main cfg is used for the creation of the simulation hash/ uid)